# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/nsICookieManager.idl
# Commit: 5e078f860dfb
# Full Hash: 5e078f860dfb9368bd747e4a8c53d07c7e8809ee
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-13 04:28:43
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 2 - CookieStore API - IPDL, r=edgul,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D215145
# ==============================================================================

diff -r d55b6525d7b2 -r 5e078f860dfb netwerk/cookie/nsICookieManager.idl
--- a/netwerk/cookie/nsICookieManager.idl	Thu Sep 12 16:48:18 2024 +0000
+++ b/netwerk/cookie/nsICookieManager.idl	Thu Sep 12 16:48:19 2024 +0000
@@ -229,6 +229,11 @@
   Array<nsICookie> getCookiesFromHost(in AUTF8String aHost,
                                       in jsval aOriginAttributes);
 
+  [notxpcom]
+  nsresult getCookiesFromHostNative(in AUTF8String aHost,
+                                    in OriginAttributesPtr aOriginAttributes,
+                                    out Array<nsICookie> aCookies);
+
   /**
    * Returns an array of all cookies whose origin attributes matches aPattern
    *
